mender-update/cli/cli.cpp

```diff
@@ -55,12 +55,6 @@ const cli::Command cmd_install {
 				.description =
 					"Return exit code 4 if a manual reboot is required after the Artifact installation",
 			},
-			// TODO: not implemented
-			cli::Option {
-				.long_option = "passphrase-file",
-				.description =
-					"Passphrase file for decrypting an encrypted private key. '-' loads passphrase from stdin",
-			},
```

Comment:
> Changelog: Deprecate `mender install --passphrase-file` CLI option. This option was added by mistake in previous versions of the Mender client and had no use (private key is not required to install an Artifact).

I think this should also be rephrased, it makes it sound too serious. How about: "Remove `--passphrase-file` argument from command scopes where it has no use. Move it to `mender-auth daemon --passphrase-file`, which is the only appropriate location."

Reply:
Thanks. It is applicable to `daemon` and `bootstrap`, though.